posts/2020-07-31-Rust-1.45.2.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Announcing Rust 1.45.2"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.45.2. Rust is a
```

Comment:
I'm not sure how I feel about the wording of "happy to announce". In a way, the release is a result of previous mistakes, so perhaps it's not something we should be celebrating?

Reply:
I think this is fine - we are in some sense happy to fix the bug at least. If you have alternative suggestions though I'd be happy to consider them.

Reply:
> The Rust team announces...

Reply:
> > The Rust team announces...

I was also considering this general direction, though I had the impression that the correct tense would be "The Rust team is announcing". I'm not a native speaker, though :upside_down_face:

Reply:
Pushed an update.